docs/tutorials/kicad.md

```diff
@@ -0,0 +1,170 @@
+
+## What is a footprint?
+
+In KiCad and in the context of electronic design, a footprint refers to the physical layout or pattern on a printed circuit board (PCB) that corresponds to a specific electronic component's pins and package. Each electronic component, such as resistors, capacitors, integrated circuits, etc., has a specific physical shape and arrangement of pins or pads that need to be accurately represented on the PCB.
+
+The footprint includes information about the component's pin locations, pad sizes, and other details necessary for correctly placing and soldering the component onto the PCB. Creating or selecting the appropriate footprint is crucial for ensuring that the physical components fit correctly onto the PCB and align with the electrical connections specified in the schematic.
+
+Footprints are associated with symbols in the design process. While symbols are used in schematic capture to represent the logical or electrical connections between components, footprints are used in the layout phase to represent the physical layout on the PCB.
+
+## What is a 3D model?
+
+In KiCad, a 3D model refers to a three-dimensional representation of a component or object used in electronic design, specifically for printed circuit board (PCB) layout and visualization. 3D models in KiCad are associated with the footprints of electronic components. These 3D models provide a realistic view of how the physical component will appear on the PCB, aiding in the design and verification process.
+
+## Installation 
+
+You can get KiCad from [here](https://www.kicad.org/download/) for your OS.
+
+## How to add custom schematics, footprints and 3D models
+
+KiCad comes with a standard library that has a wide variety of components. For most project you will only need to import just a bunch of specific integrated circuits or custom components. We will work with a *Raspberry Pi Pico W*, a development board designed around RP2040 microcontroller(MCU). KiCad's standard library has the RP2040 designs, but for the lab work we need to import the development board symbol, footprint and 3D model, as it is not included in the standard library.
+
+### Getting the Raspberry Pi Pico W KiCad files
+
+We can get the symbol, footprint and 3D model of the development board from this git [repository](https://github.com/bdubs-astro/Raspberry_Pi_Pico_W_Kicad_Files.git).
+
+Create a directory where you want to store your downloaded design files and clone the repository there.
+
+```
+git clone https://github.com/bdubs-astro/Raspberry_Pi_Pico_W_Kicad_Files.git
+```
+
+### Import the symbol
+
+Open the Symbol Libraries manager by clicking `Preferences` -> `Manage Symbol Libraries...`.
+
+<div align="center">
+![Symbol Libraries](images/symbol_libraries.png)
+</div>
+
+Click the `+` icon and give your symbol a nickname and the path to the `.kicad_sym` file.
+
+<div align="center">
+![Symbol Libraries Import](images/symbol_libraries_import.png)
```

Comment:
I suggest highlighting the imported line, otherwise it is hard to see the difference from the previous image.